travis.sh

```diff
@@ -108,6 +108,25 @@ install () {
     opam depext arakoon.1.8.12 orocksdb.0.2.1
     opam install ${OPAM_DEPENDS}
 
+    echo "Installing etcd based arakoon"
```

Comment:
Arakoon 1.8.12 will already be installed by this point ... and then we'll do it again with the exact version we want, so we'll be wasting travis resources (and feedback time) with the current flow.
Remove arakoon from the OPAM_DEPENDS env var so this doesn't happen? (May require some more fiddling than that though, not sure...)